magento/pwa-studio
# [PWA-733] Backmerge release/7.0 into develop

# ❗ DO NOT SQUASH MERGE THIS PR ❗ 

## Description

Instead of waiting another month until we actually merge the release branch into main, and then merge that into develop, we should go ahead and back merge early to realize bug and last minute features merged directly into the release branch. Some things have come up that will need corrected in that back merge:

- Make use of new design tokens, specifically with the `venia-error` color.
- Scrub version bumps so develop remains at 6.0.0 (I'm not sure this is required, but makes sense IMO).

## Related Issue





* [[PWA-733](https://jira.corp.magento.com/browse/PWA-733)] Backmerge release/7.0 into develop

## Acceptance 



### Verification Stakeholders

### Specification



### Verification Steps


There is no new code in this PR, so we really just need to run critical path smoke tests to verify nothing got auto-merged incorrectly.

## Screenshots / Screen Captures (if appropriate)

## Checklist

* I have updated the documentation accordingly, if necessary.
